tests/test_coreg/test_affine.py

```diff
@@ -36,14 +102,53 @@ class TestAffineCoreg:
         inlier_mask=inlier_mask,
         transform=ref.transform,
         crs=ref.crs,
-        verbose=False,
+        verbose=True,
```

Comment:
still needed or was for testing/debugging?

Reply:
I think we can leave as is, because `pytest` only returns prints when the test fails, so it won't pollute the output and is always useful even for later.